layouts/partner/azure.html

```diff
@@ -78,7 +78,7 @@ <h2>Creating a simple Azure Function</h2>
                         <a class="btn inline-block my-2" href="{{ relref . "/docs/get-started" }}">Get Started</a>
                     </p>
                     <p>
-                        Pulumi can be used on any resource covering serverless, containers, and infrastructure using JavaScript, TypeScript, Python, and Go.
+                        Pulumi can be used on any resource covering serverless, containers, and infrastructure data using JavaScript, TypeScript, Python, and Go.
```

Comment:
The word `data` doesn't quite work in this context.

```suggestion
                        Pulumi can be used on any resource covering serverless, containers, and infrastructure using JavaScript, TypeScript, Python, and Go.
```

Reply:
Totally fine to remove